website/versioned_docs/version-23.0/WatchPlugins.md

```diff
@@ -148,3 +148,35 @@ class MyWatchPlugin {
   }
 }
 ```
+
+## Customization
```

Comment:
@SimenB I added it under the `version-23.0` is this OK? Should I add it under a new different version folder instead? Given that it is not present in `23.0`